README.md

```diff
@@ -68,20 +66,20 @@ This can be specifically useful for server rendering when you want the datetime
 
 #### `formatter` (optional)
 A function that takes four arguments:
-  - value : An integer value, already rounded off
-  - unit : A string representing the unit in english. This could be one of:
+  - `value` : An integer value, already rounded off
+  - `unit` : A string representing the unit in english. This could be one of:
     - 'second'
     - 'minute'
     - 'hour'
     - 'day'
     - 'week'
     - 'month'
     - 'year'
-  - suffix : A string. This can be one of
+  - `suffix` : A string. This can be one of
     - 'ago'
     - 'from now'
-  - epochSeconds: The result of `Date.now()` or the result of a custom `now` prop.
-  - nextFormatter: A function that takes no arguments and gives you the result of the defaultFormatter using the same arguments above.
+  - `epochMiliseconds`: The result of `Date.now()` or the result of a custom `now` prop.
```

Comment:
`Date.now()` returns [miliseconds](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now) - what do you think about renaming this to match the [formatter code](https://github.com/nmn/react-timeago/blob/master/src/index.js#L22)?